### PR TITLE
openshift/kubernetes: add optional adhoc cgroupsv2 test

### PR DESCRIPTION
**Pull request by vrutkovs**
cgroupsv2 verification test:
* a new step to configure nodes (set `systemd.unified_cgroup_hierarchy=1` kernel arg)
* workflow to use the step on AWS
* optional test for openshift/kubernetes

Currently build tests are failing due to [BUILD-278](https://issues.redhat.com/issues/BUILD-278) (see https://github.com/openshift/builder/pull/246)

### CODE REVIEW

**Review by gabemontero — COMMENTED** [review]
how about update the 4.9 config/presubmit files as well @vrutkovs 

otherwise, the openshift/builder stuff lgtm, however I am not an approver in the openshift/builder OWNERS file, so we'll still need @adambkaplan 's signoff

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Comment by adambkaplan** [comment]
/lgtm cancel

Yes we should have the jobs configured for the 4.9 release branch as well.

**Comment by vrutkovs** [comment]
I think it would be best to split builder and kubernetes tests to get approvals sooner, I'll create a new PR

**Pull request closed without merging**
